test/spec/SearchParameters/removeXFacet.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var SearchParameters = require('../../../src/SearchParameters');
+
+describe('removeDisjunctiveFacet', function() {
```

Comment:
We can perhaps add a test with the default `state = new SearchParameters()`.

Reply:
added empty parameters tests